.ibm/pipelines/validate.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Can you please add some context about this script?
1. What is the script supposed to do?
2. When should someone modify this script?
3. Anything else that you think might help someone modify or read this script in the future.

Reply:
@dharmit can you check the changes I have added small comments/context .